### DISCUSSION TITLE
how can I use WYSIWYG when editing page

### DISCUSSION THREAD

**Discussion by Zhengyueewee2e** [outline/outline#4290] [General] [answered]
I wan't to use markdown to edit page ,how can i change to WYSIWYG

**Comment by Limezy** [comment, marked as answer]
Which version are you running ? 
The current editor is already WYSIWYG

**Comment by almereyda** [comment]
To second the previous post, the editor is a WYSIWYG (What you see is what you get.) editor (the second part of your sentence). Your page will be rendered as Markdown when you download your document via the menu in the top right.

Also see:

- https://github.com/outline/outline/discussions/2233

Are you instead asking for a [WYSIWYM (What you see is what you mean.)](https://en.wikipedia.org/wiki/WYSIWYM) editor that allows you to write markdown (the first part of your sentence), and that should then be displayed as HTML?

If you [searched the discussions here for `Markdown`](https://github.com/outline/outline/discussions?discussions_q=markdown), you could find these previous discussions on that case:

- https://github.com/outline/outline/discussions/3326
- https://github.com/outline/outline/discussions/1570

In the last one, we also find this answer:

- https://github.com/outline/outline/discussions/1570#discussioncomment-84812
  >  we've considered a raw mode several times, it might still happen but such a thing is incompatible with a lot of the more advanced features that we hope to build – so enabling it would come at quite a cost.
